iotfunctions/anomaly.py

```diff
@@ -322,11 +305,14 @@ def execute(self, df):
             # one dimensional time series - named temperature for catchyness
             temperature = dfe[[self.input_item]].fillna(0).to_numpy().reshape(-1,)
 
-            logger.debug('Spectral: ' + str(entity) + ', ' + str(self.input_item) + ', ' + str(self.windowsize) + ', ' +
+            logger.info('Spectral: ' + str(entity) + ', ' + str(self.input_item) + ', ' + str(self.windowsize) + ', ' +
```

Comment:
debug over info. Add info only if  you think the user may make some mistake in this stage.